wallet/src/params.rs

```diff
@@ -43,6 +45,8 @@ impl Default for Params {
             genesis_hash: Hash::default(),
             genesis_prev_hash: Hash::default(),
             sync_address_batch_length: 10,
+            max_vt_weight: 20_000,
+            max_dr_weight: 80_000,
```

Comment:
Is it possible to set them as constants from data_structures? Same as `INPUT_SIZE`, etc...

Reply:
they are consensus constants, so I suppose that I have to read from other part, because INPUT_SIZE is a harcoded constant

Reply:
it is only used for test